types/http-errors/http-errors-tests.ts

```diff
@@ -1,86 +1,74 @@
-
 import * as createError from 'http-errors';
 import * as express from 'express';
 
-var app = express();
-
-declare global {
-    namespace Express {
-        export interface Request {
-            user?: any
-        }
-    }
-}
+const app = express();
 
-app.use(function (req, res, next) {
-    if (!req.user) return next(createError(401, 'Please login to view this page.'));
+app.use((req, res, next) => {
+    if (!req) return next(createError('Please login to view this page.', 401));
     next();
 });
 
 /* Examples taken from https://github.com/jshttp/http-errors/blob/1.3.1/test/test.js */
```

Comment:
Sorry, I was not very clear (in fact, not at all).

When [I wrote the definition](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/4808/files#diff-599e1f561d1de6e517617b8a79e98175), in order to be sure to cover the different syntaxes offered by http-errors, I copy-pasted every call `create(...)` from https://github.com/jshttp/http-errors/blob/1.3.1/test/test.js inside http-errors-tests.ts.
([http-errors README.md](https://github.com/jshttp/http-errors/blob/master/README.md) were not detailed enough).

At the time it was version 1.3.1, now it is version 1.6.2 => would be nice to do the same to ensure everything is covered by the definition and then update the comment: `/* Examples taken from https://github.com/jshttp/http-errors/blob/1.3.1/test/test.js */` => `/* Examples taken from https://github.com/jshttp/http-errors/blob/1.6.2/test/test.js */`